accesshandler/pkg/providers/arg-schemas.go

```diff
@@ -0,0 +1,27 @@
+package providers
+
+import "github.com/invopop/jsonschema"
+
+// GetArgumentTitleFromSchema will lookup an argument in a provider arg schema and return the title if it exists else and empty string
+func GetArgumentTitleFromSchema(s *jsonschema.Schema, argumentID string) string {
```

Comment:
I found this json schema to be imperfect for this usecase, We could consider making the Args/ArgSchema a more rigid type to make this easier